mapie/tests/test_regression.py

```diff
@@ -441,3 +443,35 @@ def test_pred_loof_isnan() -> None:
         k=0,
     )
     assert len(y_pred) == 0
+
+
+def test_pipeline_compatibility() -> None:
```

Comment:
Shall we test this pipeline against a Mapie model that uses a custom numpy array containing the preprocessed data ?

Reply:
No need for preprocessed data, scikit-learn pipeline do activate all intermediate `transform` steps at prediction time.